minimesos/src/main/java/com/containersol/minimesos/main/Main.java

```diff
@@ -3,153 +3,125 @@
 import com.beust.jcommander.JCommander;
 import com.beust.jcommander.Parameter;
 import com.beust.jcommander.Parameters;
-import com.containersol.minimesos.MesosCluster;
-import com.containersol.minimesos.MinimesosException;
-import com.containersol.minimesos.cmdhooks.CliCommandHookExecutor;
-import com.containersol.minimesos.marathon.Marathon;
-import com.containersol.minimesos.mesos.*;
-import com.github.dockerjava.api.DockerClient;
-import org.apache.commons.lang.StringUtils;
+import com.containersol.minimesos.cluster.ClusterRepository;
+import com.containersol.minimesos.cluster.MesosCluster;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.TreeMap;
-
 /**
  * Main method for interacting with minimesos.
  */
 @Parameters(separators = "=", commandDescription = "Global options")
 public class Main {
-    private static final Logger LOGGER = LoggerFactory.getLogger(Main.class);
 
-    private static CommandUp commandUp;
-
-    private static String clusterId;
+    private static final Logger LOGGER = LoggerFactory.getLogger(Main.class);
 
-    @Parameter(names = {"--help", "-help", "-?", "-h"}, help = true)
+    @Parameter(names = {"--help", "-help", "-?", "-h"}, description = "Show help")
     private static boolean help = false;
```

Comment:
Holly Molly!! It took two hours to discover the `static` word, which was killing all tests !!!